src/generators/vcf_file_generator.py

```diff
@@ -36,14 +36,14 @@ class VcfFileGenerator:
 ##INFO=<ID=transcriptLevelConsequence,Number=.,Type=String,Description="VEP consequence of the variant on the Transcript">
 ##INFO=<ID=geneImpact,Number=1,Type=String,Description="Variant impact scale for Gene">
 ##INFO=<ID=transcriptImpact,Number=.,Type=String,Description="Variant impact scale for Transcript">
-##INFO=<ID=alleleSymbol,Number=1,Type=String,Description="The human readable name of the Allele">
+##INFO=<ID=allele_symbols,Number=.,Type=String,Description="The human readable name of the Allele">
 ##INFO=<ID=soTerm,Number=1,Type=String,Description="The Sequence Ontology term for the variant">
 ##INFO=<ID=alleles,Number=.,Type=String,Description="The alleles of the variant">
 ##INFO=<ID=allele_of_gene_ids,Number=1,Type=String,Description="The gene ids that the Allele is located on">
 ##INFO=<ID=allele_of_gene_symbols,Number=1,Type=String,Description="The gene names that the Allele is located on">
 ##INFO=<ID=allele_of_transcript_ids,Number=.,Type=String,Description="The gene ids that the Allele is located on">
 ##INFO=<ID=allele_of_transcript_gff3_ids,Number=.,Type=String,Description="The transcript gff3ID that the Allele is located on">
-##INFO=<ID=symbol_text,Number=1,Type=String,Description="Another human readable representation of the allele">
+##INFO=<ID=allele_symbols_text,Number=.,Type=String,Description="Another human readable representation of the allele">
```

Comment:
this needs to match whatever you put below otherwise the VCF parser will explode